src/engine/srv_internal.h

```diff
@@ -195,6 +195,8 @@ struct dss_xstream *dss_get_xstream(int stream_id);
 int dss_xstream_cnt(void);
 void dss_mem_total_alloc_track(void *arg, daos_size_t bytes);
 void dss_mem_total_free_track(void *arg, daos_size_t bytes);
+void
```

Comment:
Please maintain consistency in formatting.

Reply:
I can't. It is what clang-format does. I don't like it either.